### ISSUE TITLE
Update development.html with latest cobrapy changes

### ISSUE THREAD

**Issue by matthiaskoenig** [biosustain/cameo#132]
I just read the 
http://cameo.bio/development.html

and found some out of date things due to the new cobrapy release. 
Things like

>Solver interface
Cameo deviates from cobrapy in the way optimization problems are solved by using a separate solver interface provided by the optlang package (see optlang_interface). The following benefits ....

I understood that cobrapy now also uses the optlang approach with all its advantages. This should be updated in the documentation.

**Comment by hredestig** [comment]
Indeed, cameo has pinned it's version of cobrapy to 0.5.11 and we are working on refactoring cameo to the recent changes. We'll update this when cobrapy 0.6.0 has been released.
